### DISCUSSION TITLE
New starts for replays

### DISCUSSION THREAD

**Discussion by jerith** [endless-sky/endless-sky#8742] [Ideas]
I've recently played through ES several times (to varying degrees) trying out different playstyles and looking for fun alternatives to what I usually do in the early to mid game. One of the things I noticed is that several factions have ships that would be interesting to fly in the early game, but by the time you can acquire them they'd be a step backwards. The most obvious of these are the Hai Aphid, Grasshopper, and Scarab, but the Coalition have a nice set as well and a hyperdrive-equipped Smew or Petrel in Remnant space could be fun to try.

I've started putting together a plugin containing some custom starts, unlockable after reaching various parts of the story. At the moment only the Hai start is actually implemented (you get an Aphid, 50k credits, and no debt) but I plan to do a Remnant version with a custom-outfitted Smew or Petrel and maybe something in Human space as well. (Anything that needs a jump drive to get back to Human space would be problematic, so I'm skipping them for now.) All of these will have the tutorial disabled and the relevant first contact missions (if any) completed, but will otherwise be as close to the default start as I can manage. Unlike @Hurleveur's advanced Hai start in #7721, my goal is to provide some early-game variety for later playthroughs without skipping any of the existing content.

Is this something that's worth opening a PR for, or would it be better to have as a plugin?

**Comment by Saugia** [comment]
This would likely be plug-in related for the foreseeable future. Currently, our plan with alternate starts, once we get to them, is to have a few separate humans starts - say, you grew up on a Paradise Planet, or on a Dirt Belt planet, and start out with a slightly different sum. That's something Derpy and others plan to add someday. The Hai one is separate from this as, as you say, it's for a bit of a different time in the game. We probably won't be doing much of that, especially because it's a lot of work to try and work that into a specific period of the campaign.

Aside from this, if we are going to do alternate starts, it would be starts within reason, and pertaining to the game's story. It wouldn't make sense if you're a Hai, or a Remnant, growing up, as the game's story is told from the perspective of you being a merchant human growing up and the history happening around that.

**Comment by Hurleveur** [comment]
I didnt make the Hai start skip content because I really wanted it that way, but because changing the campaigns to react to it fully would be too much work, given our standards.
That is especially true for HR; and it could still be done later once that campaign stabilizes.

We'll encounter the same issue if we want a certain level of depth to them
Btw the remnant start is already planned by @Zitchas iirc

**Comment by jerith** [comment]
Oops, I seem to have forgotten an important piece of information above. My Hai start's protagonist isn't a character who grew up in Hai space. They're a native of Prime who joined a merchant crew and ended up there, buying a ship of their own as soon as they can afford it. The idea is to fit in with existing assumptions about the character's knowledge and background in the game while still getting to play with the small and (relatively) cheap Hai ships at the start.

Doing something similar for my Remnant start would be more difficult, but that's much more of a personal experiment and I don't expect it to ever be in the base game. Remnant culture isn't really compatible with the kind of independent entrepreneurship the player needs, and their available ships reflect that to some degree. I'm also not sure if it'll even be fun to play.

Finally, since I started this discussion I've been thinking about alternate starts in Human space. These would probably work better with a little more story attached to them than I'd originally planned. For example, an unwilling pirate who escapes their "employer" in a stolen ship and tries to become an independent merchant. (That one might be unlocked by helping Joe and Maria, and/or the pirate rescue boarding mission. I haven't looked at any of proposed (or existing) pirate content to see if something along those lines is already there, though.)

Something else I forgot to mention is that if I do PR these, it would be as individual starts rather than a collection of all of them.

**Comment by Zitchas** [comment]
For the minimalist side of things, my Lampyrid Start plugin does that. ( https://github.com/Zitchas/ES_Lampyrid_Start )I should probably update the description to mention it, but in addition to the lampyrid starts, it has a variety of other start options. These were designed to fit into the existing game with minimal-to-no alterations to existing stories or player backstory references, so in every case the player is from New Boston, but ended up on some other world before the game starts. Likewise, the player always has some connection to the textile industry, but the nature of it changes a little. For instance, in the Paradise Worlds start, the player was a logistics manager for a textile company instead of a regular worker on the mill floor. As such, they start off the game closing out their investment account to get their starting cash instead of getting a loan. 

- Paradise world (easy start)
- Deep world (alternate start)
- South world (hard start)
- New Boston with a lampyrid (with regular starting credits)
- New Boston with a lampyrid (start with only 10k, no loan, and start a little earlier)
- New Boston with a lampyrid (very early)

**Reply by Zitchas** [reply]
Just to note: the "start earlier" comments mean that the player starts at an earlier date. I've found that the bombing that starts off the FW conflict, not to mention all the early talk about the conflict and why it's happening, all have a lot more impact if I've actually been to any of the places involved or had the opportunity to experience the contrast between flying around the south and flying around the paradise worlds.

Even just an extra 6 months improves things a lot, but I generally prefer a year or so.

**Reply by jerith** [reply]
Nice! I'll definitely give those a try.

**Comment by Hurleveur** [comment]
The advanced Hai start used to have vanilla background as well, but it wasnt different enough iirc so I gave it proper Hai background.

It just depends what we want as starts for vanilla, but simple ones could just remain plugins (but officialy supported?)

**Comment by Saugia** [comment]
In any case, the planned vanilla start additions that have been discussed and are being considered are different planet areas within human space and slightly different backgrounds. The way missions, jobs, etc. are worded throughout the game (including in some other alien areas) come from the perspective you're a human native to human space, grew up there, and ventured from there.

Being a human growing up in Hai space is plausible with the right edits and the right set-up, but is a bit of work.

I would say something like a Remnant human start will be much more difficult because of how much dialogue in human space won't make sense if you've originated from the Remnant, but I don't doubt Zitchas's ability to write it in, though it would definitely be something for after we've completed Navy/Syndicate stories as well as the Remnant one being far along. 

Anything else would be plug-ins - starts that make you non-human. The game's story isn't built around the player being a non-human character.
